index.html

```diff
@@ -934,6 +943,13 @@ <h3>Processing Model</h3>
 surface resource fetch events.</p>
 <div class="issue" data-number="27"></div>
 <ol data-link-for="PerformanceResourceTiming">
+<li>If the resource's <a data-cite="Fetch#concept-request">Request</a>'s
+<a data-cite="Fetch#concept-request-destination">destination</a> equals to
+"document", "object" or "embed", and was not triggered by 
+<a data-cite="HTML#process-the-iframe-attributes">process the iframe attributes</a>,
+<a data-cite="HTML#process-the-object-attributes">process the object attributes</a>, or
+<a data-cite="HTML#process-the-embed-attributes">process the embed attributes</a>, 
```

Comment:
Hmm, I don't think iframe should be special here.

Reply:
The reasoning behind it is that the top level document should have visibility to iframe loads it initiated (or that were initiated in its context), but should not have visibility into iframe loads initiated by the iframe's content. I'm open to other ways to define the above.

/cc @bzbarsky

Reply:
Oh, that seems somewhat of a mismatch with how fetch groups should work. Also, what if I set `contentWindow.location`, I guess that doesn't count?

Reply:
Right, the intent is to exclude navigations due to `location` sets, link clicks in the iframe, etc, etc.

I'm not 100% sure how fetch groups should relate to this.  Which fetch group is the load for an `<iframe src="stuff">` supposed to be in?

Reply:
@bzbarsky my rough idea was that each global (client) has its own fetch group and since iframe always creates a global it'd have its own (embed and object are weird here and would require some special casing of sorts; my idea was that if they end up creating a global we'd move the fetches).

Reply:
That makes sense to me.  In that world, the load for  an `<iframe src="foo">` would go in the  fetch group of the iframe's `about:blank` global.  But for resource timing purposes we want to  add it to the resource timing of the parent page.

Reply:
Yeah, I think the way resource timing does its tracking is somewhat unfortunate as we cannot hang it on this concept. We end up with another table in already complex (and undefined) landscape. Basically requires more special code paths to be carefully vetted each time we touch things.

Reply:
I think we should be able to hang resource timing on the **client** of the Request.  Would that work?

Reply:
I don't think so. With `<iframe src=...>` the `about:blank` global is the client (and with which the service worker is associated).

Reply:
> With `<iframe src=...>` the about:blank global is the client

In this case we start with  https://html.spec.whatwg.org/multipage/iframe-embed-object.html#process-the-iframe-attributes then go to https://html.spec.whatwg.org/multipage/iframe-embed-object.html#otherwise-steps-for-iframe-or-frame-elements which does a https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate and says:

> Any navigation required of the user agent in the process the iframe attributes algorithm must use the iframe element's node document's browsing context as the source browsing context.

When we land in the navigate algorithm, we end up in https://html.spec.whatwg.org/multipage/browsing-the-web.html#process-a-navigate-fetch which does:

> Set request's client to sourceBrowsingContext's active document's relevant settings object

How does the client end up as that `about:blank` global?  What am I missing?

Reply:
Okay, I'm a little skeptical this is all agreed upon across implementations, but maybe this is all mostly correct and then it would probably work, indeed.